coding-concepts.js

```diff
@@ -16,44 +16,45 @@ const colors = ["tangerine", "magenta", "lilac", "daffodil"]
 // --------------------1) What will this log?
 
 const cohort = "LEARN 2022"
-// console.log(cohort.length)
+console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: 10
+// b) Verify and explain: 10 was the output. I counted the amount of charaters and spaces in the string
 
 
 // --------------------2) What will this log?
 
 const greeting = "Hello World!"
-// console.log(greeting[4])
+console.log(greeting[4])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: error
+// b) Verify and explain: o was the output. i expected an error because thought [4] was only for arrays. [4] counted from 0-4 inside the string
```

Comment:
For sure!  Arrays and strings have a lot of similarities, such as they both use zero indexing and we can use bracket notation on both to retrieve a specific value or character at a given index.